packages/react-components/README.md

```diff
@@ -73,6 +84,8 @@ export function WalletConnectorButton(props: Props) {
 }
 ```
 
+It's important that the `ConnectorType` reference passed to the hook is fixed.
```

Comment:
Is it important enough to throw an error if it's not? Can also fix it by wrapping the parameter in a 

```
const { current: connectorType } = useRef(paramConnectorType);
```

and log a warning if `paramConnectorType` changes.

Reply:
It's a good suggestion but I'm unsure if it's the correct solution in the grander scheme of things. If you follow the instructions to put the connection types in global constants (or refs) then you won't run into any problems. Of course you never know, but I don't see a reason why any reasonably competent react dev would pass fresh instances on each render. And if they did, it would be immediately visible that the buttons didn't render selection correctly.

I think that if we checked this here then there would be other places where we should do it too. And doing that for detecting an issue that I don't really expect to happen is overkill IMO.

Reply:
Well, can we ensure the competence level of the developers using the module? 😄 It was also phrased as a question as I don't know the consequences of passing a fresh connector type. If you think it's not required, then it's fine by me, but I think it's good to help to stop users doing weird things by throwing an error if you know something leads to unexpected behavioiur.

Reply:
Hehe only one way to find out ;) Passing a new instance doesn't really break anything other than `===` comparisons which is used by e.g. `isSelected`. So I think the answer to your question is no, it's not important enough.

I guess you could also imagine that a button could change role for whatever reason. It might be weird, but could happen.

We can always add this checking later if anyone ever gets bitten by it. Less is more until then :)

Reply:
Sure. 👍